Automattic/wp-calypso
# Blocks: Add JetpackConnectHeaderLogo

As part of an alternative to #24605, this PR adds a new block called `JetpackConnectHeaderLogo` that handles the logic for displaying the default Jetpack logo or a co-branded logo.

To test:

- Checkout PR
- Go to http://calypso.localhost:3000/devdocs/blocks/jetpack-connect-header-logo
- Run tests with `npm run test-client client/blocks/jetpack-connect-header-logo`

For visuals, see http://calypso.localhost:3000/devdocs/blocks/jetpack-connect-header-logo.

cc @Automattic/luna for some feedback.

I'd like to add some height/width props to this as well.
